### ISSUE TITLE
No idea what to do to get token to get started with import for the quickstarter.

### ISSUE THREAD

**Issue by iconoclasthero** [harness/gitness#3492]
So i'm following the quick start guide and I get to the point where I want to do something, i.e., import a repo from my github.  I get this with zero context of what to do next.  How do I get this token?  

**please show me what to do next to get started with your application** specifically, what do i do to get this token.  incidentally, i'm really frustrated at this point because iw ent looking for a way to generate an oauth token for github and it's impossible to find.  i really hate getting to stoping points like this because i know i'll never get back to this project if i don't make progress here and now.  you really need to provide the quick-start user with all the context necessary to rip and roll getting this up and running.

**Comment by fucksophie** [comment]
More than sure it's one of these: https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens

**Comment by bradrydzewski** [comment]
That is correct, the `token` refers to your GitHub personal token, should you want to import a repository.  You can create your personal token in your GitHub account settings (see https://www.youtube.com/watch?v=FugonDV6RdE)

Note that you don't _need_ to use the Import button to add an existing repository to Gitness — this is purely available for convenience.  Similar to GitHub / GitLab / Bitbucket, you can also just create a new repository in Gitness and then `git push` your code to it. The instructions for this would be displayed in the user interface at the time of repository creation.

**Comment by bradrydzewski** [comment]
>  you really need to provide the quick-start user with all the context necessary to rip and roll getting this up and running.

Please note that we do provide a [Quick Start](https://docs.gitness.com/) guide in our documentation.  In fact, the home page of the Gitness documentation is the Quick Start.

<img width="1157" alt="Screenshot 2024-04-01 at 9 58 34 AM" src="https://github.com/harness/gitness/assets/817538/0f4aea7b-0dc3-4f38-b223-75e2ee3370b6">

And the documentation does explicitly instruct our users to create a personal access token, with a link to the GitHub documentation, which has detailed [instructions](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#creating-a-personal-access-token-classic) for token creation.

<img width="1034" alt="Screenshot 2024-04-01 at 9 51 17 AM" src="https://github.com/harness/gitness/assets/817538/8276bc76-9136-435d-9989-6ed4861344e1">

I understand that it can be frustrating when you struggle to use a product. This is honestly the first time anyone has struggled with this particular feature, and it does appear that we have documented this feature quite well. If you continue to feel frustrated with the product and are having difficulty with our documentation, please consider using our Slack channel as an alternative https://docs.gitness.com/support
